scripts/do-release.sh

```diff
@@ -204,6 +209,9 @@ function update_operator_pr() {
     done
 
     sed -i -E "s/(.*Version +=) .*/\1 \"${release['version']#v}\"/" "projects/${project}/apis/submariner/v1alpha1/versions.go"
+    if [[ "${release['pre-release']}" != "true" && -n "${release['bundle']}" ]]; then
```

Comment:
```suggestion
    if [[ "${release['pre-release']}" != "true" ]]; then
```

Since you're already validating that the bundle field is set if it's not a pre release